README.md

```diff
@@ -391,7 +391,7 @@ Browse [builtin actions](./pkg/plugins/builtin)
 |**`ssh`**     | Connect to a remote system and run commands on it | [Access plugin doc](./pkg/plugins/builtin/ssh/README.md)
 |**`email`**   | Send an email | [Access plugin doc](./pkg/plugins/builtin/email/README.md)
 |**`ping`**    | Send a ping to an hostname *Warn: This plugin will keep running until the count is done* | [Access plugin doc](./pkg/plugins/builtin/ping/README.md)
-
+|**`script`**    | Execute a script under `scripts` folder *Warn: This plugin will keep running until execution is done* | [Access plugin doc](./pkg/plugins/builtin/script/README.md)
```

Comment:
I don't understand the warning. There is a timeout option, and this is not any more true than with other plugins e.g. HTTP ?

Reply:
This warn is the same as `ping` plugin because it's not just about ms, it can be several seconds.
But maybe you right, I should remove it.